### ISSUE TITLE
DeprecationWarning: invalid escape sequence

### ISSUE THREAD

**Issue by stephenfin** [pymupdf/PyMuPDF#482]
## Bug description

Calling `Page.getText('blocks')` on PDFs that contain invalid Python escape sequences (e.g. `\ `) result in the following warnings:

    ../fitz/fitz.py:5404: DeprecationWarning: invalid escape sequence '\ '
      return _fitz.TextPage_extractBLOCKS(self, lines)

This is a warning now but [may or may not be an error in Python 3.10](https://mail.python.org/archives/list/python-dev@python.org/thread/ZX2JLOZDOXWVBQLKE4UCVTU5JABPQSLB/).

## To Reproduce (mandatory)

1. Create the following test script and save as `test.py`:

        import sys
        import fitz

        pdf = fitz.open(sys.argv[1])
        for page in pdf.pages():
            page.getText('blocks')

2. Save the attached file locally

3. Run the script against the file with [deprecation warnings enabled](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONWARNINGS):

        PYTHONWARNINGS=d python3 test.py test_aafigure.pdf

## Expected behavior (optional)
The strings should be marked as rawstring (e.g. `r'\ '`) internally or escaped.

## Screenshots (optional)
N/A

## Your configuration (mandatory)
 - Fedora 31 (64 bit)
 - Python 3.7.6
 - PyMuPDF 1.16.16, **wheel**

```
3.7.6 (default, Jan 30 2020, 09:44:41) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] 
 linux 
```

```
PyMuPDF 1.16.16: Python bindings for the MuPDF 1.16.0 library.
Version date: 2020-03-29 09:44:30.
Built for Python 3.7 on linux (64-bit).
```

## Additional context (optional)
I did try to fix this myself, but I haven't worked with SWIG (or Python bindings to a C lib) before and got lost. Sorry :disappointed:

**Comment by JorjMcKie** [comment]
Thanks for reporting this! ... And for your interest in PyMuPDF.
Let me have a look. I'll be back 😎

**Comment by JorjMcKie** [comment]
Where is the attached PDF please?

**Comment by stephenfin** [comment]
Whoops, sorry. Attached.

[test_aafigure.pdf](https://github.com/pymupdf/PyMuPDF/files/4431304/test_aafigure.pdf)

**Comment by JorjMcKie** [comment]
ok, thanks, will look at it

**Comment by JorjMcKie** [comment]
Fixed it I think by using ``PyUnicode_DecodeRawUnicodeEscape`` instead of ``PyUnicode_DecodeUnicodeEscape``.

**Comment by stephenfin** [comment]
Awesome. Thanks! Let me know if you need anything from me testing wise. I assume the reproducer I provided did the trick.

**Comment by JorjMcKie** [comment]
> I assume the reproducer I provided did the trick.

Yes, thanks again. Your observation also did occur for the "text", "words", "(x)html" and "(raw)dict" variants of ``getText()``.
Until fairly recently, I just used ``PyUnicode_FromStringAndSize`` to make Python strings from extracted PDF text. But I learned from user provided PDF examples, that this text is **not** reliably UTF-8 encodable. So I had to switch ... and switch again now 😉.
